serf/serf.go

```diff
@@ -25,7 +25,7 @@ import (
 // version to memberlist below.
 const (
 	ProtocolVersionMin uint8 = 2
-	ProtocolVersionMax       = 4
+	ProtocolVersionMax       = 5
```

Comment:
I'm still not sure why we would want to stop defaulting to the latest version here; there shouldn't be any changes that cause incompatibilities between clients running protocol v4 vs v5, so why keep the default at 4?